ci/fix_wheels.py

```diff
@@ -0,0 +1,55 @@
+import os
+import shutil
+import struct
+import sys
+import zipfile
+
+try:
+    _, wheel_path, dest_dir = sys.argv
+    # Figure out whether we are building on 32 or 64 bit python
+    is_32 = struct.calcsize("P") * 8 == 32
+    PYTHON_ARCH = "x86" if is_32 else "x64"
+except ValueError:
+    # Too many/little values to unpack
+    raise ValueError(
+        "User must pass the path to the wheel and the destination directory."
+    )
+# Wheels are zip files
+if not os.path.isdir(dest_dir):
+    print(f"Created directory {dest_dir}")
+    os.mkdir(dest_dir)
+shutil.copy(wheel_path, dest_dir)  # Remember to delete if process fails
+wheel_name = os.path.basename(wheel_path)
+success = True
+exception = None
+repaired_wheel_path = os.path.join(dest_dir, wheel_name)
+with zipfile.ZipFile(repaired_wheel_path, "a") as zipf:
+    try:
+        # TODO: figure out how licensing works for the redistributables
+        base_redist_dir = (
+            f"C:/Program Files (x86)/Microsoft Visual Studio/2019/"
```

Comment:
Should maybe consider using a package to do this unless we have in house expertise on Windows libraries. `delvewheel` might be an option

Reply:
Last time I tried(~1 year back), delvewheel was doing creating extra directories in the repaired wheels. 

Note: We test the pandas wheels on the windowsservercore docker images, which does not have the visual c++ redistributables. Any missing DLLS should fail the job there.